File: src/Example3Java/EX21.java
package Example3Java;

public class EX21 {
    public static void main(String[] args) {
        String[] a = {"a", "b", "c", "d", "e"};
        String[] b = {"a", "b", "c", "d", "e"};
        System.out.println("Array a == Array b : " + (a.length == b.length));
        if (a.length == b.length)
            for (int i = 0; i < a.length; i++)
                System.out.println("a[" + i + "] == b[" + i + "] : " + (a[i] == b[i]));
    }
}

File: src/Example3Kotlin/EX21.kt
object EX21 {
    @JvmStatic fun main(args: Array<String>) {
        val a = arrayOf("a", "b", "c", "d", "e")
        val b = arrayOf("a", "b", "c", "d", "e")
        println("Array a == Array b : ${a.size == b.size}")
        if (a.size == b.size)
            for (i in a.indices)
                println("a[$i] == b[$i] : ${a[i] == b[i]}")
    }
}
